app/scripts/templates/sign_in.mustache

```diff
@@ -19,9 +19,13 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
-        {{#isMigration}}
+      {{#isSyncMigration}}
           <div class="info nudge">{{#t}}Migrate your sync data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
-        {{/isMigration}}
+      {{/isSyncMigration}}
+
+      {{#isAmoMigration}}
+          <div class="info nudge">{{#t}}Migrate your Mozilla Add-ons data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
Is there a way to simplify this string? We have a string that we reuse for different services where `serviceName` is replaced. Would `{{#t}}Migrate your %(migrationName)s data by signing in to your Firefox&nbsp;Account.{{/t}}` make sense?